portal_multicurrency_totals/README.rst

```diff
@@ -16,13 +16,13 @@ To use this module, just go to the portal  and open a document list; you will se
 When accessing quotations or sale orders, a total will be displayed per
 currency:
 
-    .. image:: portal_multicurrency_totals/static/description/totals_in_so.png
+    .. image:: /portal_multicurrency_totals/static/description/totals_in_so.png
```

Comment:
Why this change?

Please check if this image is being rendered correctly on GitHub.

Reply:
I changed it because the image was being shown as a broken link in the previous version.